templates/default/testconnector-launch.sh.erb

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+help() {
+    echo ""
+    echo "usage: $0 [input_json_filepath image_name]"
+    echo ""
+    exit 1
+}
+
+if [ "$#" != "2" ] || [ "$1" == "-h" ] ; then
+  help
+fi
+
+REGISTRY_URL='<%= @registry_addr %>'
+FILENAME=$(basename "$1")
+FOLDER=$(dirname "$1")
+# TODO image name as parameter?
+IMAGE="$2"
+
+if docker run --rm -v $FOLDER:/tmp $REGISTRY_URL/$IMAGE:latest /tmp/$FILENAME; then
```

Comment:
we should probably have a directory in `/tmp/testconnector` or something.
But wait, we agreed in the design log to write it to the staging dir in a subdirectory, in the backend you have it right `/srv/hops/staging`?

Reply:
yes we are using the staging dir `/srv/hops/staging` as the work directory in hops. The `/tmp` was only for the container destination directory. Now I have changed that to `/home` as `/tmp is not created by default